src/theme-preval.js

```diff
@@ -1,5 +1,6 @@
 // @preval
 // This file needs to be a JavaScript file using CommonJS to be compatiable with preval
+// Cache bust: 2021-11-04 12:00:00 GMT (This file is cached by our deployment tooling, update this timestamp to rebuild this file)
```

Comment:
TIL

Reply:
There has to be a better way of doing this, but I did not spend more than 5 minutes researching vercel's caching mechanism